devito/ir/clusters/analysis.py

```diff
@@ -192,3 +193,34 @@ def _callback(self, clusters, d, prefix):
             return
 
         return TILABLE
+
+
+class Skewing(Detector):
```

Comment:
A lot of duplicate code. Currently when sth is TILABLE, it is SKEWABLE as well.
Though this may change.